### PR TITLE
Add context to log errors

### PR DESCRIPTION
**Pull request by romainneutron**

### CODE REVIEW

File: src/SensioLabs/Connect/OAuthConsumer.php [hunk 1]
```diff
@@ -84,36 +84,36 @@ public function getAuthorizationUri($callbackUri)
...
 
-        $this->logger->debug(sprintf("Response of AccessToken: %s", $response));
+        $this->logger->debug(sprintf('Response of AccessToken: %s', $response));
 
         $content = $response->getContent();
         $response = json_decode($content, true);
 
         if (null === $response) {
-            $this->logger->error('Received non-json response.');
+            $this->logger->error('Received non-json response.', array('response' => $content));
```
Comment (lyrixx) [inline src/SensioLabs/Connect/OAuthConsumer.php]: Did you try it ? Cause the response is an object IIRC. May be you should cast it to string.

Comment (romainneutron) [inline src/SensioLabs/Connect/OAuthConsumer.php]: Did you notice I'm not passing the response object but content.
AFAIK, after re-reading buzz source code, the output of `$response->getContent()` is a string.

Comment (lyrixx) [inline src/SensioLabs/Connect/OAuthConsumer.php]: oups ;) perfect.

**Comment by lyrixx** [comment]
Thank you Romain.

**Comment by lyrixx** [comment]
v4.2.1 released

**Pull request closed without merging**